engine/src/main/java/com/sandboni/core/engine/Processor.java

```diff
@@ -129,16 +135,16 @@ private Builder getBuilder(Context context) {
             Instant finish = Instant.now();
             log.debug("....Finders execution total time: {}", Duration.between(start, finish).toMillis());
             start = Instant.now();
-            connectors.parallelStream().filter(c-> c.proceed(context)).forEach(c -> c.connect(context));
+            connectors.parallelStream().filter(c -> c.proceed(context)).forEach(c -> c.connect(context));
             finish = Instant.now();
             log.debug("....Connectors execution total time: {}", Duration.between(start, finish).toMillis());
-        } else if (isRunAllExternalTests() && isIntegrationStage()){
+        } else if (isRunAllExternalTests() && isIntegrationStage()) {
             log.info("Running All External Tests");
             finders.parallelStream().forEach(f -> f.findSafe(context));
             return new Builder(context, FilterIndicator.ALL_EXTERNAL);
         } else { //only cnfg files
             log.info("Found changes: {}", context.getChangeScope());
-            log.info(" ** configuration file(s) were changed; All tests will be executed ** ");
+            log.info(" ** configuration files or files outside of Sandboni scope were changed; All tests will be executed ** ");
```

Comment:
can we make this msg print the exact reason why it was skipped? meaning w/o the 'or'

Reply:
Not right now because this value is taken from proceed() method that only returns a boolean. The refactoring is little more complicated. I think that we can easily check what was the root reason just looking at the files that were changed in the log.